examples/ui-demo/src/state/index.tsx

```diff
@@ -86,8 +98,35 @@ export function ConfigSync() {
     classes.forEach((x) => root.classList.add(x));
 
     const uiConfig = convertDemoConfigToUiConfig(config);
+
+    const externalWalletSection = uiConfig.auth?.sections
+      .find((x) => x.some((y) => y.type === "external_wallets"))
+      ?.find((x) => x.type === "external_wallets") as
+      | Extract<AuthType, { type: "external_wallets" }>
+      | undefined;
+
+    if (externalWalletSection?.walletConnect) {
+      const walletConnectAuthConfig = externalWalletSection?.walletConnect;
+      const walletConnectParams = {
+        ...walletConnectAuthConfig,
+        qrModalOptions: {
+          ...walletConnectAuthConfig.qrModalOptions,
+          themeVariables: {
+            "--wcm-z-index": "1000000",
+            ...walletConnectAuthConfig.qrModalOptions?.themeVariables,
+          },
+        },
+      } as WalletConnectParameters;
+
+      if (!connectors.some((x) => x.type === "walletConnect")) {
+        console.log(walletConnect(walletConnectParams));
+        // @ts-ignore
+        connectors.push(walletConnect(walletConnectParams)(wagmiConfig));
+      }
+    }
+
     updateConfig(uiConfig);
-  }, [config, updateConfig]);
+  }, [config, connectors, updateConfig]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'wagmiConfig'. Either include it or remove the dependency array.
<!-- __reviewdog__:ChA0YjBhYTU1MTliMGNhNGFhEgZyZGpzb24= -->